docs/source/science/NISAR/OPERA Surface Displacement.ipynb

```diff
@@ -0,0 +1,4440 @@
+{
```

Comment:
<p>This should be done with Xarray. The next step after that is to show cloud-optimized access patterns. <a href="https://guide.cloudnativegeo.org/cloud-optimized-netcdf4-hdf5" target="_blank">https://guide.cloudnativegeo.org/cloud-optimized-netcdf4-hdf5/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>